docs/quickstart.rst

```diff
@@ -122,9 +122,7 @@ After downloading the dataset and unzipping it in your current working directory
     ]]
     y = data["riders"]
 
-The machine learning workflow is the art of creating *model selection triples*, a combination of features, algorithm, and hyperparameters that uniquely identifies a model fitted on a specific data set. As part of our feature selection, we want to identify features that have a linear relationship with each other, potentially introducing covariance into our model and breaking OLS (guiding us toward removing features or using regularization). We can use the Rank2D_ visualizer to compute Pearson correlations between all pairs of features as follows:
-
-.. _Rank2D: http://www.scikit-yb.org/en/latest/api/yellowbrick.features.html#module-yellowbrick.features.rankd
+The machine learning workflow is the art of creating *model selection triples*, a combination of features, algorithm, and hyperparameters that uniquely identifies a model fitted on a specific data set. As part of our feature selection, we want to identify features that have a linear relationship with each other, potentially introducing covariance into our model and breaking OLS (guiding us toward removing features or using regularization). We can use the :ref:`Rank2D <:doc:`Rank2D <api/features/rankd>`: Rank2D>`_ visualizer to compute Pearson correlations between all pairs of features as follows:
```

Comment:
Hi again @Naba7! This still isn't quite right -- would you please replace the line that reads: 

`We can use the :ref:`Rank2D <:doc:`Rank2D <api/features/rankd>`: Rank2D>`_ visualizer to compute Pearson correlations between all pairs of features as follows:`
with 
`We can use the :doc:`api/features/rankd` visualizer to compute Pearson correlations between all pairs of features as follows:`

Restructured text definitely takes a bit of practice -- since you're planning to be a regular contributor, it might be a good idea to read up on [Sphinx](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)! Here's the documentation for how to do [internal relative cross references in rst](http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-doc).